fabtests/common/shared.c

```diff
@@ -3059,6 +3064,25 @@ void ft_parse_hmem_opts(int op, char *optarg, struct ft_opts *opts)
 	}
 }
 
+void parse_opts_range (char* optarg)
+{
+       int start, inc, end;
+       int i, range_test_cnt;
+       sscanf (optarg, "r:%d,%d,%d", &start, &inc, &end);
```

Comment:
remove space after function name.  Add check for return value to validate that input was correct and return a failure code if not.